tests/driver/nodejs/build.rs

```diff
@@ -29,6 +29,9 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         "cargo:rustc-env=SLINT_NODE_NATIVE_LIB={}",
         target_dir.join(nodejs_native_lib_name).display()
     );
+    println!(
+        "cargo:rustc-env=SLINT_ENABLE_EXPERIMENTAL_FEATURES=1",
```

Comment:
why do we need this if the test is anyway skipped?

Also, we set the env in nodejs.rs